src/nvim/vvars.lua

```diff
@@ -0,0 +1,869 @@
+local M = {}
+
+M.vars = {
+  argv = {
+    type = 'string[]',
+    desc = [=[
+      The command line arguments Vim was invoked with.  This is a
+      list of strings.  The first item is the Vim command.
+      See |v:progpath| for the command with full path.
+    ]=],
+  },
+  char = {
+    desc = [=[
+      Argument for evaluating 'formatexpr' and used for the typed
+      character when using <expr> in an abbreviation |:map-<expr>|.
+      It is also used by the |InsertCharPre| and |InsertEnter| events.
+    ]=],
+  },
+  charconvert_from = {
+    type = 'string',
+    desc = [=[
+      The name of the character encoding of a file to be converted.
+      Only valid while evaluating the 'charconvert' option.
+    ]=],
+  },
+  charconvert_to = {
+    type = 'string',
+    desc = [=[
+      The name of the character encoding of a file after conversion.
+      Only valid while evaluating the 'charconvert' option.
+    ]=],
+  },
+  cmdarg = {
+    type = 'string[]',
+    desc = [=[
+      The extra arguments ("++p", "++enc=", "++ff=") given to a file
+      read/write command.  This is set before an autocommand event
+      for a file read/write command is triggered.  There is a
+      leading space to make it possible to append this variable
+      directly after the read/write command. Note: "+cmd" isn't
+      included here, because it will be executed anyway.
+    ]=],
+  },
+  collate = {
+    type = 'string',
+    desc = [=[
+      The current locale setting for collation order of the runtime
+      environment.  This allows Vim scripts to be aware of the
+      current locale encoding.  Technical: it's the value of
+      LC_COLLATE.  When not using a locale the value is "C".
+      This variable can not be set directly, use the |:language|
+      command.
+      See |multi-lang|.
+    ]=],
+  },
+  cmdbang = {
+    desc = [=[
+      Set like v:cmdarg for a file read/write command.  When a "!"
+      was used the value is 1, otherwise it is 0.  Note that this
+      can only be used in autocommands.  For user commands |<bang>|
+      can be used.
+    ]=],
+  },
+  completed_item = {
+    desc = [=[
+      Dictionary containing the most recent |complete-items| after
+      |CompleteDone|.  Empty if the completion failed, or after
+      leaving and re-entering insert mode.
+      Note: Plugins can modify the value to emulate the builtin
+      |CompleteDone| event behavior.
+    ]=],
+  },
+  count = {
+    type = 'integer',
+    desc = [=[
+      The count given for the last Normal mode command.  Can be used
+      to get the count before a mapping.  Read-only.  Example: >vim
+        :map _x :<C-U>echo "the count is " .. v:count<CR>
+      <
+      Note: The <C-U> is required to remove the line range that you
+      get when typing ':' after a count.
```

Comment:
```suggestion
      get when typing `:` after a count.
```
(or ":", but not single quotes)

Reply:
Isn't single quotes fine for a single char?

Reply:
> Isn't single quotes fine for a **single** char?

Ah, you're right, I missed that (also for non-alphanumeric chars); I was worried that the vimdoc parser is not as lenient, but I just tested that it is.

(Have I mentioned how much I hate vimdoc and it's Pirate Code of a "specification"?)